weave/trace_server/object_class_util.py

```diff
@@ -0,0 +1,151 @@
+from typing import Any, Optional, TypedDict
```

Comment:
this is an edited version of the now deleted `weave/trace_server/base_object_class_util.py`

Reply:
mostly just comments and name changes